blazor/dialog/predefined-dialogs.md

```diff
@@ -0,0 +1,338 @@
+---
+layout: post
+title: Predefined dialog in Blazor Dialog Component | Syncfusion
+description: Check out and learn all about Predefined dialog in Syncfusion Blazor Dialog component and much more here.
+platform: Blazor
+control: Dialog
+documentation: ug
+---
+
+# Predefined dialogs in the Blazor Dialog component
+
+The dialog component is used to render the `Alert`, `Confirm`, and `Prompt` dialogs with minimal code. The alert, confirm, and prompt dialogs are shown using [DialogServices](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialogService.html).
+
+## Configuration
+
+### Blazor Server App
+
+* For **.NET 6** app, open the **~/Program.cs** file and import Syncfusion.Blazor.Popups.
```

Comment:
SfDialogService -> not explained about registering service. importing alone enough? don't we have to register.